icu4c/source/test/intltest/strcase.cpp

```diff
@@ -763,7 +766,40 @@ void StringCaseTest::TestDutchTitle() {
                 testOptions
             );
         }
-        
+    }
+
+    // Bug ICU-21919
```

Comment:
You don't really need this when you have the additional test case line above on line 746, do you?

Reply:
Done. No, it's not necessary.